cms/wp/db/backup/download.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * Download the latest database backup file.
+ *
+ * @file
+ */
+
+declare(strict_types=1);
+
+namespace Deployer;
+
+task('cms:wp:db:backup:download', static function (): void {
+    $latestBackup = run('\ls -rt -1 {{backups}} | tail -1');
```

Comment:
Is that `\` required here?

Reply:
It is a thing I do to prevent inheritance.  Typically 'ls' is aliased and I don't want those aliases to get in the way. I'm not sure what the shell command for deployer uses and which dotfiles it sources on shell creation (and thus aliases created).  So in a sense the slash is not necessary but it also doesn't hurt - and guarantees that aliases don't get in the way.

Reply:
Got it, thanks for the explanation, and good idea!